lib/command.ts

```diff
@@ -75,10 +85,10 @@ export function command(
       return;
     }
 
-    // allow a module to be provided instead of separate builder and handler
+    // Allow a module to be provided as builder, rather than function:
     if (isCommandBuilderDefinition(builder)) {
       self.addHandler(
-        [cmd].concat(aliases),
+        [cmd].concat(aliases) as HandlerArrayItem[],
```

Comment:
I think this may avoid the `as`:
```suggestion
      const placeholder: HandlerArrayItem[] = [cmd].concat(aliases);
      self.addHandler(
        placeholder,
```

Reply:
I managed to tighten up the types quite a bit 👍 

This turned out to be a good audit of the command API.